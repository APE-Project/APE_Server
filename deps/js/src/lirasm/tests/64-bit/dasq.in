q = immq 12345
d = qasd q
q2 = dasq d
one = immd 1.0      ; do some intermediate stuff to make it less trivial
two = immd 2.0
three = addd one two
i2 = q2i q2
reti i2
