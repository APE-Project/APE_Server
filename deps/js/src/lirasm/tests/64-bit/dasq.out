Output is: 12345
